rs/protobuf/def/state/canister_state_bits/v1/canister_state_bits.proto

```diff
@@ -474,8 +474,7 @@ message CanisterStateBits {
   reserved "log_visibility";
   // Log visibility for the canister.
   LogVisibilityV2 log_visibility_v2 = 51;
-  // The capacity of the canister log in bytes.
-  uint64 log_memory_limit = 56;
+  reserved 56;
```

Comment:
FYI: Even though it was not actually used by anyone, but the field was populated in system_state with the default value, so I reserve this tag number not to be used in the future.